packages/serverless-workflow-vscode-extension/README.md

```diff
@@ -1,7 +1,6 @@
-## Kogito Serverless Workflow Editor
+## Openshift Serverless Logic Editor
```

Comment:
@ricardozanini @domhanak Let me know if you want any changes on the extension readme. This goes on the extension page in the marketplace.